src/NServiceBus.Core/Persistence/InMemory/SagaPersister/InMemorySagaPersister.cs

```diff
@@ -13,7 +14,7 @@ class InMemorySagaPersister : ISagaPersister
     {
         public Task Complete(IContainSagaData sagaData, SynchronizedStorageSession session, ContextBag context)
         {
-            var inMemSession = (InMemorySynchronizedStorageSession) session;
+            var inMemSession = (InMemorySynchronizedStorageSession)session;
```

Comment:
I think someone did a cleanup with a wrong configuration? Pretty sure our shared configuration adds the space.